decidim-core/app/cells/decidim/content_blocks/last_activity_cell.rb

```diff
@@ -44,6 +44,10 @@ def valid_activities
 
       private
 
+      def cache_hash
+        "decidim/content_blocks/last_activity/#{Digest::MD5.hexdigest(valid_activities.map(&:updated_at).to_s)}"
```

Comment:
same as with the preivous cell

Reply:
> same as with the preivous cell

In this case, suppose that the number of valid activities increases to a large number, which means a long identification key for the cache system to find. With an MD5 hash, we make sure that the key will remain the same length.